src/parsers/srdf.hxx

```diff
@@ -247,19 +247,25 @@ namespace pinocchio
               if (joint_tag.first == "joint")
               {
                 std::string joint_name = joint_tag.second.get<std::string>("<xmlattr>.name");
-                const Scalar joint_config = (Scalar)joint_tag.second.get<double>("<xmlattr>.value");
                 typename Model::JointIndex joint_id = model.getJointId(joint_name);
-
+                
                 // Search in model the joint and its config id
                 if (joint_id != model.joints.size()) // != model.njoints
                 {
                   const JointModel & joint = model.joints[joint_id];
-                  assert(joint.nq() == 1 && "SRDF:half_sitting only handles 1 DoF joints");
-                  model.neutralConfiguration(joint.idx_q()) = joint_config; // joint with 1 dof
-                  // model.neutralConfiguration.segment(joint.idx_q(),joint.nq()) = joint_config; // joint with more than 1 dof
+                  typename Model::ConfigVectorType joint_config(joint.nq());
+                  if(joint.nq() == 1)  joint_config[0] = (Scalar)joint_tag.second.get<double>("<xmlattr>.value");
+                  else
+                  {
+                    const std::string joint_val = joint_tag.second.get<std::string>("<xmlattr>.value");
+                    std::istringstream config_string(joint_val);
+                    const std::vector<double> config_vec((std::istream_iterator<double>(config_string)), std::istream_iterator<double>());
+                    joint_config = ModelTpl<Scalar,Options,JointCollectionTpl>::ConfigVectorType::Map(config_vec.data(), config_vec.size());
```

Comment:
Use Model here instead of ModelTpl<Scalar,Options,JointCollectionTpl>

Reply:
I tried using typename Model::ConfigVectorType.
gcc cannot does not understand the Map attribute when I do that, but it finds the attribute when I use the full name.

Reply:
because you miss the template keyword when doing so. You can see with @jmirabel for that point if he is at LAAS today to explain you this additional info.